src/mds/MDSRank.h

```diff
@@ -151,6 +151,31 @@ class Finisher;
 class ScrubStack;
 class C_ExecAndReply;
 
+struct MDSMetaRequest {
```

Comment:
Is this really required? Cannot the MClientRequest (for the rename op to the peer mds) be enough for tracking?